src/users/users.service.ts

```diff
@@ -273,16 +274,27 @@ export class UsersService implements OnModuleInit {
   }
 
   async findByIdUserSettings(userId: string): Promise<UserSettings | null> {
```

Comment:
_findUserSettingsById_ is a better name

Reply:
The renaming of the internal functions needs more brainstorming.
It requires more effort in order to keep naming consistent.